docs/how-to-guides/isis-demos-and-workshops.md

```diff
@@ -18,6 +24,12 @@ These specific ISIS demonstrations were held on Wednesday, June 27 2012 (ISIS3.4
 
 ## **Planetary Data Workshop - Cassini ISIS Workshop - June 2012**
 ------
+
+!!! warning "The subsequent workshop could be considered outdated but retained for historical reference."
+
+    For the most current documentation for the listed applications, please consult the [ISIS Application Docs Website](https://isis.astrogeology.usgs.gov/).
+    Additionally, refer to our other [How-To Guides](../how-to-guides/) for current tutorials.
```

Comment:
The relative path `../how-to-guides` actually took me to a url with the path `<docs.url>/how-to-guides/how-to-guides` so I get a 404 when attempting to hit this hyperlink. Found that changing the relative link to `../how-to-guides/index.md` will at least take the user to the main How-To Guides page.